### ISSUE TITLE
Link to Procedure in an Article is Missing in HTML and PDF Output

### ISSUE THREAD

**Issue by taroth21** [openSUSE/suse-xsl#162]
See http://opensuse.github.io/daps/doc/cha.daps.user.features.html#sec.daps.user.basics:

"2.4 Defining Documentation Projects #
The easiest way to set up a new documentation project from scratch is to use the DAPS initialization script daps-init. For instructions how to do so, refer to . "
                                                                                               ^^^

In the XML (daps_user_concept.xml), we have:
```
For  instructions how to do so, refer to <xref linkend="pro.daps.init"/>.
```
where *pro.daps.init* is an ID in the article that belongs to the same set as *daps_user_concept.xml*

DB 4, obviously...

**Comment by sknorr** [comment]
This is an issue in the `intra.book.linking` mode and also affects SLE 12 SP1 documentation (there is a link to the ID `proc.ad.join` somewhere).

**Comment by sknorr** [comment]
@tomschr Could you take over? Assigning you tentatively. Otherwise, just unassign yourself.

**Comment by tomschr** [comment]
The above fix create a `<span class="intraxref">...</span>` element. ATM, it's not styled (see attached picture).

@sknorr please take over and create a CSS rule! :-)

![procedure-intraxref](https://cloud.githubusercontent.com/assets/1312925/11712490/cd9f0cf0-9f2d-11e5-8e57-d2e652463b34.png)

**Comment by sknorr** [comment]
This should fix the formatting issues: 4a6e6ee24d83d1de078a43dcf0cffb795e6d83f7
